src/simsopt/mhd/vmec.py

```diff
@@ -399,6 +399,10 @@ def load_wout(self):
         #self.wout.volume = f.variables['volume_p'][()]
         f.close()
 
+        self.s_full = np.linspace(0, 1, self.wout.ns)
```

Comment:
For the sake of code readability, is it possible to expand 's_full' and 's_half' to something more descriptive?